extensions/wikia/AdEngine/js/provider/paidAssetDrop.js

```diff
@@ -16,13 +16,12 @@ define('ext.wikia.adEngine.provider.paidAssetDrop', [
 	}
 
 	function fillInSlot(slotName, success, hop) {
+		var pageType = adContext.getContext().targeting.pageType;
 		log(['fillInSlot', slotName], 'debug', logGroup);
 
 		if (pad.isNowValid(adContext.getContext().opts.paidAssetDropConfig)) {
-			if (!paidAssetDropSlotLoaded) {
-				pad.injectPAD('#' + slotName, 'mobile');
-				paidAssetDropSlotLoaded = true;
-			}
+			doc.getElementById(slotName).innerHTML = '';
```

Comment:
According to http://jsperf.com/innerhtml-or-dom/4 `innerHTML` is slower than removing element from DOM and creating it again.